resources/schemas/sfdx-project.schema.json

```diff
@@ -980,12 +980,14 @@
             "items": {
                 "type": "string",
                 "enum": [
+                    "analyze",
                     "prepare",
```

Comment:
@alanjaouen we should just add these values when we supported it. it is not supported yeat the "ignoreOnStage" feature for "validate", "quickbuild", "promote" and "publish"